word_scripts/wordfind_site.py

```diff
@@ -0,0 +1,27 @@
+import requests
+import word_file_merger
+from bs4 import BeautifulSoup
+
+def extract(htmlList):
+    words = []
+    
+    finding_words = list(filter(lambda x: x.get('class') == ['defLink'], htmlList))
+    
+    for link in finding_words:
+        words.append(link.find('a').get('href')[6:-1])     
```

Comment:
you can do this in a more functional and pythonic approach using built-in `map()` function
ex `map(lambda x: x.find('a').get('href')[6:-1], finding_words)` and you could just return this